components/dash-core-components/src/components/Textarea.react.js

```diff
@@ -73,6 +73,11 @@ const Textarea = ({
     );
 };
 
+Textarea.dashPersistence = {
```

Comment:
Is this the recommended way of setting default for persistence?  
It will only work in Dash >=3 ?

Reply:
Yes, only for dash>=3 for support of the persistence props that was previously in `defaultProps` in functional components and react 18.

Reply:
Does it work in dash< 3 to have the default set in the function signature?  If not, we should check if Dash 3.0 is being use and if not use `defaultProps`?  Is the best way to check for Dash 3.0 by seeing if `window.dash_component_api` exists?

I also I get an error when trying to set default props in the function signature when the prop is a required prop by Mantine.  This might be a separate issue, but do you know how to handle that case?